### DISCUSSION TITLE
Haing issues in compiling the ZKsync contract.

### DISCUSSION THREAD

**Discussion by HasanZaigam** [Cyfrin/foundry-full-course-cu#3033] [General]
### **I'm having issues compiling the ZKsync contract.**

![Screenshot 2024-12-12 032834](https://github.com/user-attachments/assets/c4808da2-e974-4a08-94d4-b16918c371cb)

I tried it for the first time, but I encountered an error. Is anyone here to help me?

**Comment by KMean** [comment]
I can't replicate your error. Try again creating a fresh remix and make sure you're using [this contract](https://github.com/Cyfrin/remix-fund-me-cu/blob/main/zkSyncContract/FundMezkSync.sol). If you want to use the compile project button make sure your FundMe.sol is in the contracts folder.

**Reply by EngrPips** [reply]
Great Heads-up. Also, you can clear your browser cache before you retry.
